### ISSUE TITLE
add editor line range to which comment applies to comment widget's aria label

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#193500]
cc @alexr00, something we should do as a result of discussion with @jooyoungseo

**Comment by meganrogge** [comment]
this already happens when one uses the comments view to navigate👍

**Comment by meganrogge** [comment]
i think we should use `alert` or `status` to indicate this when going to next range since this info is otherwise just strictly visual.

**Comment by alexr00** [comment]
@meganrogge so the idea is the following:

1. Run command go to next/previous commenting range
2. aria `status`: "Commenting range lines 4-10"

And:

1. Focus on a comment widget
2. aria `status`: "Comment on lines 3-7" or "Comment on line 3"

**Comment by meganrogge** [comment]
I think the latter works well enough already. It's the former that I think we need.

![Screenshot 2023-09-22 at 8 22 54 AM](https://github.com/microsoft/vscode/assets/29464607/f74e9cd1-7bef-4fc0-8c3b-473d05bbb9a2)

**Comment by meganrogge** [comment]
Actually, I wonder if the precise line number is important. Given a user isn't told what line number they're on when they move their cursor, perhaps instead we could say "spans 4 lines". Since a user has invoked this command, they know that it's a comment range, so we probably want to leave out "comment/commentable" to decrease noise

**Comment by meganrogge** [comment]
I will ask Roberto to confirm my suspicion.

**Comment by jooyoungseo** [comment]
@meganrogge IMO, the line reference could be useful when users want to backtrace later.

**Comment by meganrogge** [comment]
@jooyoungseo, shall we leave out the term 'Comment' here and just say 'lines 730-736' to reduce verbosity?

**Comment by meganrogge** [comment]
Or "Lines 1 to 5 have a comment range" as a screen reader user can disrupt it mid message via escape?

**Comment by meganrogge** [comment]
I am leaning toward just "Lines 1 to 5" given the user invoked the command so knows it's a comment range already.

**Comment by alexr00** [comment]
Thanks for making the change!

**Comment by meganrogge** [comment]
@bhavyaus open an editor that has some commentable ranges. Turn on a screen reader. use the command jump to next /previous commenting range. verify you hear the range like "lines 123 to 128" as the cursor's position gets updated.

**Comment by amunger** [comment]
nothing is being announced for me with that commend, I have screen reader optimized and commenting verbosity on

**Comment by meganrogge** [comment]
@amunger depending on what you have going on otherwise, it will wait to say that since we're using `status` instead of `alert` here.

**Comment by meganrogge** [comment]
Hey @amunger are you seeing it jump to the commenting ranges?

https://github.com/microsoft/vscode/assets/29464607/97bce22f-44e5-46d4-925c-e36a83e5b3bb

**Comment by amunger** [comment]
works on my desktop, does not work on my surface laptop - neither the speech viewer nor through volume, nothing is read at all when running that command by NVDA. It is successfully navigating between commentable regions.

**Comment by meganrogge** [comment]
@amunger sounds like you've verified this works on your desktop. I'm curious to know if other aria alerts are working on your surface laptop - for example, the untitled editor hint
